vortex-array/src/array/bool/compute/mod.rs

```diff
@@ -1,28 +1,30 @@
 use crate::array::{BoolArray, BoolEncoding};
 use crate::compute::unary::{FillForwardFn, ScalarAtFn};
-use crate::compute::{AndFn, ArrayCompute, ComputeVTable, FilterFn, OrFn, SliceFn, TakeFn};
+use crate::compute::{ArrayCompute, BinaryBooleanFn, ComputeVTable, FilterFn, SliceFn, TakeFn};
 use crate::ArrayData;
 
-mod boolean;
-
 mod fill;
 pub mod filter;
 mod flatten;
 mod scalar_at;
 mod slice;
 mod take;
 
-impl ArrayCompute for BoolArray {
-    fn and(&self) -> Option<&dyn AndFn> {
-        Some(self)
-    }
+impl ArrayCompute for BoolArray {}
 
-    fn or(&self) -> Option<&dyn OrFn> {
-        Some(self)
+impl ComputeVTable for BoolEncoding {
+    fn binary_boolean_fn(
+        &self,
+        _lhs: &ArrayData,
+        _rhs: &ArrayData,
+    ) -> Option<&dyn BinaryBooleanFn<ArrayData>> {
+        // We only implement this when other is a constant value, otherwise we fall back to the
+        // default implementation that canonicalizes to Arrow.
+        // TODO(ngates): implement this for constants.
+        // other.is_constant().then_some(self)
```

Comment:
Why is this not implemented anymore?

Reply:
It is implemented by the arrow fallback in the entry-point function